lib/workers/branch/status-checks.ts

```diff
@@ -7,6 +7,25 @@ import {
   isActiveConfidenceLevel,
 } from '../../util/merge-confidence';
 
+export async function getBranchStatus(
```

Comment:
```suggestion
export async function resolveBranchStatus(
```

To avoid confusion with `platform.getBranchStatus()`